core/js/src/main/scala/zio/internal/PlatformLive.scala

```diff
@@ -40,8 +40,9 @@ object PlatformLive {
       }
 
       def reportFailure(cause: Cause[Any]): Unit =
-        if (!cause.interrupted)
+        cause.stripSilent.filterNot(_.interrupted).foreach { cause =>
```

Comment:
I think we want to use `interruptedOnly` instead of `interrupted` here and in the versions for other platforms.

Reply:
I need to merge master into my branch to fix it